source/common/res/features/import-notification/main.js

```diff
@@ -22,4 +23,3 @@
     setTimeout(poll, 250);
   }
 }());
-
```

Comment:
Oy, didn't realize I got rid of that. I'll add it back when I fix the other issue in this feature. See my comments in the related issue.